emails/tests/models_tests.py

```diff
@@ -41,6 +41,12 @@ def test_has_bad_words_with_bad_words(self):
     def test_has_bad_words_without_bad_words(self):
         assert not has_bad_words('happy')
 
+    def test_has_bad_words_exact_match_on_small_words(self):
+        assert has_bad_words('ho')
+        assert not has_bad_words('horse')
+        assert has_bad_words('ass')
+        assert not has_bad_words('cassandra')
```

Comment:
Do we need a couple more tests here for the cases where we have a 4+ letter word from the bad list? If I'm reading the code/rule correctly this should still mean that a domain like "shell" would be invalid, correct?